### PR TITLE
New Version: Astronomer.Astro version 1.8.3

### PR DESCRIPTION
**Pull request by neel-astro**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90115)

### CODE REVIEW

File: manifests/a/Astronomer/Astro/1.8.3/Astronomer.Astro.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=NVS2.CRLF.5-1-22000-832.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Astronomer.Astro
+PackageVersion: 1.8.3
+InstallerLocale: en-US
+InstallerType: portable
+InstallModes:
```
Comment (mdanish-kh) [inline manifests/a/Astronomer/Astro/1.8.3/Astronomer.Astro.installer.yaml:8]:
```suggestion
```

File: manifests/a/Astronomer/Astro/1.8.3/Astronomer.Astro.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- interactive
```
Comment (mdanish-kh) [inline manifests/a/Astronomer/Astro/1.8.3/Astronomer.Astro.installer.yaml:9]:
```suggestion
```

File: manifests/a/Astronomer/Astro/1.8.3/Astronomer.Astro.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- silent
```
Comment (mdanish-kh) [inline manifests/a/Astronomer/Astro/1.8.3/Astronomer.Astro.installer.yaml:10]:
```suggestion
```

File: manifests/a/Astronomer/Astro/1.8.3/Astronomer.Astro.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+- silentWithProgress
```
Comment (mdanish-kh) [inline manifests/a/Astronomer/Astro/1.8.3/Astronomer.Astro.installer.yaml:11]:
```suggestion
```

File: manifests/a/Astronomer/Astro/1.8.3/Astronomer.Astro.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+InstallerSwitches:
```
Comment (mdanish-kh) [inline manifests/a/Astronomer/Astro/1.8.3/Astronomer.Astro.installer.yaml:12]:
```suggestion
```

File: manifests/a/Astronomer/Astro/1.8.3/Astronomer.Astro.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  Silent: --silent
```
Comment (mdanish-kh) [inline manifests/a/Astronomer/Astro/1.8.3/Astronomer.Astro.installer.yaml:13]:
```suggestion
```

File: manifests/a/Astronomer/Astro/1.8.3/Astronomer.Astro.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+  SilentWithProgress: -silent
```
Comment (mdanish-kh) [inline manifests/a/Astronomer/Astro/1.8.3/Astronomer.Astro.installer.yaml:14]:
```suggestion
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
InstallModes and InstallerSwitches are not needed for `portable` / standalone applications. We can get rid of this redundant information in the manifest.

**Comment by neel-astro** [comment]
@mdanish-kh thanks for the review, have updated the PR with suggestion

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**